### DISCUSSION TITLE
Partition coefficient between cells and plasma

### DISCUSSION THREAD

**Discussion by Vincent-AC** [Open-Systems-Pharmacology/Forum#1156] [Questions & Problems]
Hello,

I work with mobi and I have a question about the K_cell_pls partition coefficient. When I select the rowland and rodgers formula I find that the equation is the one that rowland and rodgers use for determining K_tissue (doi :  10.1002/jps.20322 )

The issue is that K_cell_pls being the partition coefficient between intracellular and plasma should not be equal to K_tissue (which includes extracellular water in addition to intracellular), unless I misunderstand the equations in Mobi ?

Best regards,
Vincent

**Comment by StephanSchaller** [comment]
The calculated constants will not be equal as K_cell_pls will use the volume fraction from cell, while K_tissue used the fractions for cell + interstitial. Maybe the publication from [Schmitt et al. ](https://pubmed.ncbi.nlm.nih.gov/17981004/) will be of help here.

**Reply by Vincent-AC** [reply]
Oh, i did not get that point from the formula in the Allcalculationmethods file. But then it means that if I want to input experimental Kp values I have to adjust it for predicted interstitial Kint-plasma doesn't it ?

**Reply by Vincent-AC** [reply]
Also, now that I have access to my computer again, I wonder what f_ew (extracellular water fraction) is in PK-Sim when using the rodgers and rowland formula to calculate K_cell_pls, is it set to 0 then ?
